src/subscription_manager/exceptions.py

```diff
@@ -74,7 +75,14 @@ def format_bad_ca_cert_exception(self, bad_ca_cert_error, message_template):
         return message_template % bad_ca_cert_error.cert_path
 
     def format_ssl_error(self, ssl_error, message_template):
-        return message_template % str(ssl_error)
+        # SSL error has fixed form: [SSL/BIO/...: ERRORO_ID] readable string (file.c:#line)
```

Comment:
Please revert the change to this method. It's okay to show the user the full error message, as long as we give them a human-readable message too.